client/src/components/ProfilePage/index.js

```diff
@@ -65,11 +70,107 @@ const AccordionDetails = withStyles((theme) => ({
 const ProfilePage = (props) => {
   const [expanded, setExpanded] = React.useState("panel1");
   const { user } = React.useContext(UserContext);
-  // console.log(user);
   const classes = useStyles();
+  
+  const [listings, setListings] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [error, setError] = useState(false);
+
+  const fetchItems = async () => {
+    setLoading(true);
+    const itms = await getUserItems(user._id);
+    if (itms.success) {
+      setListings(itms.listings);
+      setError(false);
+    } else setError(true);
+    setLoading(false);
+  };
+
+  useEffect(() => {
+    fetchItems();
+  }, []);
+
+  const StyledText = styled.h1`
+  background-image: linear-gradient(#2196F3,#21CBF3);
+  font-size: 2.5rem;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  margin: 0;
+  margin-left: 1rem;
+`;
+
+  const goToItem = (id) => props.history.push(`/item/${id}`);
+  
   const handleChange = (panel) => (event, newExpanded) => {
     setExpanded(newExpanded ? panel : false);
   };
+
+  //if user has no listings
+  if(listings.length == 0)
```

Comment:
Pls use === instead of ==
=== checks for true equality and wont perform type-casting unlike == so === is much safer to use

Reply:
fixed